src/static/api_documentation.yml

```diff
@@ -78,6 +93,21 @@ paths:
         '503 - Service Unavailable':
           $ref: '#/components/responses/NoCeleryWorkers'
 
+  "/models/{scenarioId}/getPollution":
+    get:
+      summary: Retrieves information on pollutant load for buildings, roads, and car parks, for a given MEDUSA model output id.
+      description: |-
+        It is recommended to use the geoserver API if it is possible, since this is a proxy around that to expose the functionality to all users.
+      parameters:
+        - $ref: '#/components/parameters/ScenarioId'
+      responses:
+        '200 - OK':
+          $ref: '#/components/responses/MedusaOutput'
+        '404 - Not Found':
+          $ref: '#/components/responses/ScenarioNotFound'
```

Comment:
This ScenarioNotFound mentions flood models, but you can make it more generic and just remove "flood"